multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/pizzareader/PizzaReaderGenerator.kt

```diff
@@ -16,6 +16,7 @@ class PizzaReaderGenerator : ThemeSourceGenerator {
         SingleLang("Phoenix Scans", "https://www.phoenixscans.com", "it", className = "PhoenixScans", overrideVersionCode = 4),
         SingleLang("GTO The Great Site", "https://reader.gtothegreatsite.net", "it", className = "GTO", overrideVersionCode = 3),
         SingleLang("LupiTeam", "https://lupiteam.net", "it", overrideVersionCode = 3),
+        SingleLang("TuttoAnimeManga", "https://tuttoanimemanga.net", "it", overrideVersionCode = 1)
```

Comment:
```suggestion
        SingleLang("TuttoAnimeManga", "https://tuttoanimemanga.net", "it", overrideVersionCode = 4)
```

Needs to be `4` so the new version is higher, and the users receive the update.